oxd-common/src/main/java/org/gluu/oxd/common/ExpiredObject.java

```diff
@@ -93,10 +123,21 @@ public void setType(ExpiredObjectType type) {
         this.type = type;
     }
 
+    public String getTypeString()
```

Comment:
minor but would be good to format it, `{` should not be in new line

Reply:
Done